docs/pages/tooling.js

```diff
@@ -162,12 +162,64 @@ Visit the [release log](/whats_new) or check the [codemod directory](https://git
           </Box>
         </MainSection.Subsection>
         <MainSection.Subsection
-          badge="beta"
           title="Visual Studio Code extension for Gestalt"
-          description="Get Gestalt component snippets right on your VSCode editor!"
+          description="Access Gestalt component snippets and documentation right on your VSCode editor. Right at your fingertips!"
         >
+          <Text weight="bold">Quick access to Gestalt</Text>
           <Flex maxWidth={MAX_WIDTH} direction="column" alignItems="center">
             <Box marginBottom={6}>
+              <Flex direction="column">
+                <Box as="figure" width={400}>
+                  <Image
+                    alt=""
+                    color="white"
+                    naturalHeight={1036}
+                    naturalWidth={1228}
+                    src="https://i.ibb.co/G2PMw6f/Screen-Shot-2022-01-27-at-4-24-36-PM.png"
+                  />
+                  <Text size="sm" align="center">
+                    <Box as="figcaption" marginTop={3}>
+                      Quick access to Gestalt components and documentation on the VSCode sidebar.
+                      Check for the Gestalt logo on the sidebar.
+                    </Box>
+                  </Text>
+                </Box>
+                <Box as="figure" width={400}>
+                  <Image
+                    alt=""
+                    color="white"
+                    naturalHeight={978}
+                    naturalWidth={1398}
+                    src="https://i.ibb.co/9Vjhyrm/Screen-Shot-2022-01-27-at-4-45-42-PM.png"
+                  />
+                  <Text size="sm" align="center">
+                    <Box as="figcaption" marginTop={3}>
+                      {`List Gestalt components by typing <`}
```

Comment:
nit: it's not clear that we're referring to the "<" character. Perhaps quotes would make it more clear? Or styling as code?

Reply:
DONE